go/oasis-net-runner/fixtures/fixtures.go

```diff
@@ -0,0 +1,70 @@
+// Package fixtures provides network configuration fixtures.
+package fixtures
+
+import (
+	"encoding/json"
+	"math"
+
+	flag "github.com/spf13/pflag"
+	"github.com/spf13/viper"
+
+	"github.com/oasislabs/oasis-core/go/common"
+	"github.com/oasislabs/oasis-core/go/oasis-test-runner/oasis"
+)
+
+const (
+	cfgFile = "fixture.file"
+)
+
+var (
+	// Flags are command line flags for the fixtures.
+	Flags = flag.NewFlagSet("", flag.ContinueOnError)
+
+	// DefaultFlags are  command line flags for the fixture.default.* flags.
+	DefaultFlags = flag.NewFlagSet("", flag.ContinueOnError)
```

Comment:
Consider moving this variable to `default.go` as well. To make it less confusing maybe rename it to `DefaultFixtureFlags` (and the above one could be `FileFixtureFlags`?).